pilot/pkg/model/service.go

```diff
@@ -623,12 +623,15 @@ func (s *Service) GetServiceAddressForProxy(node *Proxy) string {
 // security.istio.io/tlsMode if set. Do not return Enums or constants
 // from this function as users could provide values other than istio/disabled
 // and apply custom transport socket matchers here.
-func GetTLSModeFromEndpointLabels(labels map[string]string) string {
+func GetTLSModeFromEndpointLabels(labels map[string]string, serviceaccount string) string {
```

Comment:
can you please update the comment to indicate that it also looks at "service account"?

Reply:
ok